### PR TITLE
Feature/pagination

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Adding pagination to the table.

#### What problem is this solving?
No pagination.

#### How should this be manually tested?
Checkout and upload file over 10 lines.

#### Screenshots or example usage
screen shot 2017-09-14 at 11 29 03

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export const globalVars = {
```
Comment (klzns) [inline src/constants.js]: Não é bem `globalVars`, ta mais pra valores defaults.

Comment (kevinch) [reply src/constants.js]: Pode ser, vou atualizar.

**Comment by klzns** [comment]
Acho que não tem a ver ter paginação nesse componente. 

Exibir os dados é fora do escopo dele.

**Comment by kevinch** [comment]
acho que sim, se tiver uns 1000 linhas no csv, não tem que ter paginação?

**Comment by klzns** [comment]
Ve essa issue https://github.com/vtex/react-upload-csv/issues/3

File: src/components/Form.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ class Form extends React.Component {
     const file = event.target.files[0]
     // this.props.fileChanged && this.props.fileChanged()
```
Comment (omninando) [inline src/components/Form.js:41]: commented code

**Review by omninando — CHANGES_REQUESTED** [review]

**Comment by kevinch** [comment]
Objective change. ref: https://github.com/vtex/react-upload-csv/issues/3

**Pull request closed without merging**